File: backend/src/main/java/com/lawencon/elearn/dto/attendance/AttendanceInsertResDto.java
package com.lawencon.elearn.dto.attendance;

public class AttendanceInsertResDto {
	private AttendanceInsertDataResDto data;
	private String message;
	public AttendanceInsertDataResDto getData() {
		return data;
	}
	public void setData(AttendanceInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}

File: frontend/src/app/dto/attendance/attendance.insert.res.dto.ts
import { AttendanceInsertDataResDto } from "./attendance.insert.data.res.dto";

export class AttendanceInsertResDto{
    private _data! : AttendanceInsertDataResDto
	private _message! : string

    public get data(): AttendanceInsertDataResDto {
        return this._data;
    }

    public set data(data: AttendanceInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}